dcos/packagemanager.py

```diff
@@ -41,6 +41,8 @@ def check_for_cosmos_error(*args, **kwargs):
             logger.debug("Error: {}".format(response.json()))
             error_msg = _format_error_message(response.json())
             raise DCOSException(error_msg)
+        elif (response.status_code / 100) != 2:
+            raise DCOSHTTPException(response)
```

Comment:
How does this look to the user? I.e. can I see an example of this error?

Reply:
It looks like this

```
Error while fetching [http://jlariosmu-elasticl-15r2m2ribeead-755510712.us-west-2.elb.amazonaws.com/package/describe]: HTTP 504: "Gateway Time-out".
```